src/plugins/messageLogger/index.tsx

```diff
@@ -197,7 +207,7 @@ export default definePlugin({
         return cache;
     },
 
-    shouldIgnore(message: any) {
+    shouldIgnore(message: any, is_edit: boolean = false) {
```

Comment:
```suggestion
    shouldIgnore(message: any, isEdit: boolean = false) {
```

keep casing consistent! in javascript we use camelCase